### DISCUSSION TITLE
Amazon motors vs. Vybronics - which lasts longer

### DISCUSSION THREAD

**Discussion by pdbuzzboard** [pdbuzzboard/pdbuzzboard#4] [General]
This question has come up several times.  My intuition is that the more expensive motors _should_ last longer.  I am building a test rig that will have 4 motors (2 cheap, 2 Vybronics) and activate them for 100ms in order (1,2,3,4).  This will provide 100ms on and 300ms off for each motor.  A separate Arduino will measure vibrations from each motor and note the time of failure.

While 2 is not a representative sample, I am hoping it will provide some quantitative data on life expectancy of the motors.

If anyone has other motors that they are considering, let me know and I can add them to the mix.

**Comment by twolips** [comment]
Cool idea - looking forward to it.
Arnd

**Comment by teh-teckstein** [comment]
I just built a rig with the cheap Amazon Motors, it has already run fine for four hours and I am tracking the time it has run in an excel sheet. If you would like the data to get a better idea of an MTTF where would you like it to be sent or would you just want a failure time, in hours, in the discussion board?

**Comment by pdbuzzboard** [comment]
Let's post results here.  The broader the community participation, the faster we will progress!

Thank you for the effort.

**Comment by weckstein** [comment]
My husband and sons built me a Buzz Board using your great instructions. We used the 12,000 Amazon motors. One motor stopped after 7 hours and 30 minutes of use. Two more stopped after 8 hours and 45 minutes. I have ordered the 15,000 motors to see if they have a longer life.

**Reply by PWPInnovator898** [reply]
Did the motors fail and/or or the solder joints?

**Comment by weckstein** [comment]
It was the motors that stopped. The support system for the joints is a
great idea! We replaced the 3 and all is going well. Next weekend, we are
updating system with 15,000 motors.
